packages/@okta/vuepress-site/docs/guides/migrate-to-oie/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Migrate to Okta Identity Engine
+meta:
+  - name: description
+    content: Our guide shows how to migrate your organization and clients to OIE 
+layout: Guides
+---
+
+## Enable OIE for your organization
+
+To enable Okta Identity Engine, please reach out to your account manager. If you do not have an account manager, please reach out to <oie@okta.com> for more information.
+
+## Enable `interaction code` grant
+
+Once the Okta Identity Engine feature is enabled for your org, it should become active for Okta-hosted signin flows that do not involve an OAuth application. Enabling the `interaction_code` grant type will allow OAuth applications to use the Okta Identity Engine.
+
+### Enable `interaction code` grant on an authorization server
+
+1. Navigate to `Security > API > Authorization Servers` in the Admin UI
+2. Select an authorization server and click the edit icon.
+3. Click on `Access Policies` tab.
+4. Edit `Default Policy Rule`.
+5. In the section `IF Grant type is`, click the `Interaction Code` checkbox.
```

Comment:
nit: should this be "IF" or "If"?

Reply:
In the Admin UI it appears as "IF" (it is a rule editor)